traitsui/testing/tester/locator.py

```diff
@@ -35,3 +35,17 @@ class TargetByName:
     """
     def __init__(self, name):
         self.name = name
+
+class DefaultTarget:
+    """ A locator for locating a default target.  In many cases, handlers may
+    not be registered for certain interactions on a given target.  Instead
+    there may be a solver registered a locator_class of DefaultTarget and a
+    target_class of the original target.  The registered solver will then
+    return some generic (toolkit specific) object, for which frequently used
+    handlers will be registered.  If user tries to perform the intetractions
+    of these handlers on the original object, machinery is in place for the
+    UIWrapper to try and locate a DefaultTarget and then look for the needed
+    handler automatically, if it can not find a handler directly for the
+    original target and interaction. 
```

Comment:
The above is good for developers whose tasks are implementing solvers / handlers. Thank you!  For users of the tester, this might be a bit heavy going. Most of the time, the users would not be using this object directly (it is used under the hood). Maybe we should mention this upfront. This object is going to be part of the public facing API for people _using_ the tester.